lib/l10n/app_de.arb

```
@@ -289,15 +289,15 @@
   "currency_settings_title": "Währungsauswahl",
   "currency_settings_subtitle": "Wählen Sie Ihre bevorzugten Währungen",
   "available_currencies": "Verfügbare Währungen",
-  "selected_currencies": "Ausgewählte Währungen", 
+  "selected_currencies": "Ausgewählte Währungen",
   "no_currencies_available": "Keine Währungen vom Server verfügbar",
   "select_currencies_hint": "Währungen aus der obigen Liste auswählen",
   "preview_title": "Vorschau",
   "tap_to_cycle": "Tippen, um Währungen zu wechseln",
   "calculating_text": "Berechne...",
```

RuboCop (1.86.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)